deploy/Install-MRP-app.sh

```diff
@@ -1,30 +1,31 @@
-﻿#!/bin/bash
+#!/bin/bash
+
+# Auto-accept license for Java
+echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections
 
 # Install packages
+add-apt-repository ppa:webupd8team/java
 apt-get update
-apt-get install openjdk-8-jdk -y
-apt-get install openjdk-8-jre -y
+apt-get install oracle-java8-installer -y
 apt-get install mongodb -y
 apt-get install tomcat7 -y
 apt-get install wget -y
 
-# Set Java environment variables
-export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
-export PATH=$PATH:/usr/lib/jvm/java-8-openjdk-amd64/bin
+export JAVA_HOME=/usr/lib/jvm/java-8-oracle
```

Comment:
Tomcat is sometimes picky about finding the right path. Creating a symlink helps this:
ln -s /usr/lib/jvm/java-8-oracle /usr/lib/jvm/default-java